packages/replay/src/index.ts

```diff
@@ -1,18 +1,41 @@
+/** @deprecated Use the export from `@sentry/replay` or from framework-specific SDKs like `@sentry/react` or `@sentry/vue` */
 export { replayIntegration } from './integration';
 
-export type {
-  ReplayConfiguration,
-  ReplayEventType,
-  ReplayEventWithTime,
-  ReplayBreadcrumbFrame,
-  ReplayBreadcrumbFrameEvent,
-  ReplayOptionFrameEvent,
-  ReplayFrame,
-  ReplayFrameEvent,
-  ReplaySpanFrame,
-  ReplaySpanFrameEvent,
-  CanvasManagerInterface,
-  CanvasManagerOptions,
-} from './types';
-
+/** @deprecated Use the export from `@sentry/replay` or from framework-specific SDKs like `@sentry/react` or `@sentry/vue` */
 export { getReplay } from './util/getReplay';
+
+/** @deprecated Use the export from `@sentry/replay` or from framework-specific SDKs like `@sentry/react` or `@sentry/vue` */
+export type { ReplayConfiguration } from './types';
+
+/** @deprecated Use the export from `@sentry/replay` or from framework-specific SDKs like `@sentry/react` or `@sentry/vue` */
+export type { ReplayEventType } from './types';
```

Comment:
Do JSDoc comments work like this? I think we need to add the comments to the actual declarations of the exported items.

Reply:
This made me think. We don't have to deprecate anything in v8. We can just remove it.